app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/CheckLiabilityDetailsAnswersController.scala

```diff
@@ -38,7 +38,7 @@ class CheckLiabilityDetailsAnswersController @Inject() (
 
   def displayPage(): Action[AnyContent] =
     (authenticate andThen journeyAction) { implicit request =>
-      Ok(page(request.registration, backLink))
+      Ok(page(appConfig.isPreLaunch, request.registration, backLink))
```

Comment:
We tend to DI the appConfig on the view.
Which means you then need to stub the call to `isPreLaunch` on our tests.

Reply:
That's how I was trying to do things initially but I was unable to get the tests to work: I tried overriding the application config and injecting a mock AppConfig but I could not get it to work. In the end, I liked this way as the whole AppConfig is not exposed to the view, only the value of the relevant feature switch.

Reply:
OK, I've worked out how I can test this with AppConfig injected via DI: if I construct the page object directly rather than obtaining from the DI framework then I can ensure that a mock AppConfig is used:
```
  private val formHelper       = instanceOf[formWithCSRF]
  private val govukLayout      = instanceOf[main_template]
  private val govukButton      = instanceOf[GovukButton]
  private val govukSummaryList = instanceOf[GovukSummaryList]
  private val pageTitle        = instanceOf[pageTitle]
  private val saveAndContinue  = instanceOf[saveAndContinue]
  private val mockAppConfig    = mock[AppConfig]

  private val page = new check_liability_details_answers_page(formHelper,
                                                              govukLayout,
                                                              govukButton,
                                                              govukSummaryList,
                                                              pageTitle,
                                                              saveAndContinue,
                                                              mockAppConfig
  )

```
@PauloPortugal would you prefer that I went with this approach? I don't really have strong feelings for one approach over the other.

Reply:
It would be good to be consistent with the approach.

Reply:
I have made the changes as outlined above.